app/components/samples/table_component.rb

```diff
@@ -41,7 +41,7 @@ def initialize(
     def system_arguments
       { tag: 'div' }.deep_merge(@system_arguments).tap do |args|
         args[:id] = 'samples-table'
-        args[:classes] = class_names(args[:classes], 'relative', 'overflow-x-auto')
+        args[:classes] = class_names(args[:classes], 'overflow-x-auto')
```

Comment:
**nitpick (blocking):** please revert his change

Reply:
Added back in e02db1cc5084b77a54e37854aadf8001cc5777ce